servicecontrol/powershell.md

```diff
@@ -1,19 +1,17 @@
 ---
 title: Managing ServiceControl via PowerShell
-reviewed: 2021-04-17
+reviewed: 2021-08-23
 ---
 
-
 ## ServiceControl PowerShell
 
-ServiceControl version 1.7 introduced a new graphical management utility to add, remove, update and delete instances of the ServiceControl service. These actions and some additional tools are also available via a PowerShell module called `ServiceControlMgmt`.
-
+Before the graphical management utility existed to set up ServiceControl, PowerShell actions were used to add, remove, update and delete instances of ServiceControl.
```

Comment:
I think it can be beneficial for users that have been using SC for a longer time to keep the version in here

Reply:
- We're at 4.20, at 4.x since july 2019
- Version 1.7 is from **2015**
- The management utility is _all over the place_

I thought this documentation was so old, that it was about time to remove it.